docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -136,6 +136,120 @@ tasks.named('compileKotlin', org.jetbrains.kotlin.gradle.tasks.KotlinCompilation
 </tab>
 </tabs>
 
+### Migrate from `kotlinOptions{}` to `compilerOptions{}` {collapsible="true"}
+
+Before Kotlin 2.2.0, you could configure compiler options using the `kotlinOptions{}` block. Since the `kotlinOptions{}`
```

Comment:
`kotlinOptions {}` DSL was deprecated in Kotlin 2.0.0 release. Deprecation level was raised to `ERROR` in Kotlin 2.2.0 release.

Reply:
Maybe just change `is deprecated in Kotlin 2.2.0` to `is deprecated since Kotlin 2.0.0`